### ISSUE TITLE
Node inspector crashes on latest node 0.10.36 with breakpoints

### ISSUE THREAD

**Issue by EloB** [node-inspector/node-inspector#548]
`Node Inspector v0.8.3` crashes on breakpoints or when it reaches that breakpoint I think hard to say.

Started with `node-inspector --debug-port=5859 --web-port=1337 --web-host=192.168.10.10 --no-preload`.

```
/usr/lib/node_modules/node-inspector/lib/BreakEventHandler.js:48
  var scriptId = obj.script.id,
                           ^
TypeError: Cannot read property 'id' of undefined
    at BreakEventHandler._onBreak (/usr/lib/node_modules/node-inspector/lib/BreakEventHandler.js:48:28)
    at DebuggerClient.emit (events.js:95:17)
    at DebuggerClient._emitDebuggerEvent (/usr/lib/node_modules/node-inspector/lib/DebuggerClient.js:92:8)
    at Debugger.emit (events.js:95:17)
    at Debugger._processResponse (/usr/lib/node_modules/node-inspector/lib/debugger.js:99:10)
    at Protocol.execute (_debugger.js:134:14)
    at Socket.emit (events.js:95:17)
    at Socket.<anonymous> (_stream_readable.js:765:14)
    at Socket.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:427:10)
```

**Comment by EloB** [comment]
The inspector starts and runs for a while but when it reaches that breakpoint it crashes.

**Comment by EloB** [comment]
I've cleaned my cache and localstorage. Nothing works...

**Comment by 3y3** [comment]
Looks like changes in debugger protocol.
Can you confirm this issue on 0.10.33?
Do you can reproduce this bug on all your scripts or only on a one special?

**Comment by EloB** [comment]
Everything started to break at 0.10.35 (https://github.com/joyent/node/issues/8948) then upgraded to 0.10.36. The debugger runs and after awhile it crashes on breakpoints. Have only had the problem in our code.

**Comment by 3y3** [comment]
Thanks for fast feedback.
So, it doesn't happen on all breakpoints, only on some special?
(For example I've tested my scripts last two weeks on 0.10.36 and I can't reproduce this error)

**Comment by EloB** [comment]
Yes that's correct.

**Comment by EloB** [comment]
I don't remember were in my code this happens.

**Comment by sudhir-jabong** [comment]
I have also faced same issue. It seems, Node inspector doesnt work properly on Node 0.10 version

**Comment by 3y3** [comment]
@sudhir-jabong , any example of code?

**Comment by 3y3** [comment]
Fixed in 0.10.* . Feel free to reopen it, if issue doesn't solved.
